architecture/secure_storage.rst

```diff
@@ -461,3 +494,5 @@ specification.
 .. _core/tee/tee_fs_key_manager.c: https://github.com/OP-TEE/optee_os/blob/master/core/tee/tee_fs_key_manager.c
 .. _core/tee/tee_rpmb_fs.c: https://github.com/OP-TEE/optee_os/blob/master/core/tee/tee_rpmb_fs.c
 .. _lib/libutee/: https://github.com/OP-TEE/optee_os/blob/master/lib/libutee/
+.. _mmc-tools: https://docs.kernel.org/driver-api/mmc/mmc-tools.html
```

Comment:
Refs are not case sensitive?  (`MMC-tools_` at line 346  vs `_mmc-tools` here)

Reply:
It seems it's not. I can make it _MMC-tools instead if you prefer that.